suite-native/receive/src/screens/ReceiveAccountScreen.tsx

```diff
@@ -1,87 +1,88 @@
-import { useDispatch, useSelector } from 'react-redux';
+import { useSelector } from 'react-redux';
 
-import { G } from '@mobily/ts-belt';
+import { RouteProp, useRoute } from '@react-navigation/native';
 
-import { ErrorMessage, VStack, Box } from '@suite-native/atoms';
+import { getScreenWidth } from '@trezor/env-utils';
 import {
     AccountsRootState,
-    removeButtonRequests,
-    selectAccountByKey,
-    selectSelectedDevice,
+    DeviceRootState,
+    selectDeviceAccountKeyForNetworkSymbolAndAccountTypeWithIndex,
 } from '@suite-common/wallet-core';
-import { AccountKey, TokenAddress } from '@suite-common/wallet-types';
-import { Translation } from '@suite-native/intl';
-import { ConfirmOnTrezorImage } from '@suite-native/device';
-import { Screen } from '@suite-native/navigation';
+import { BoxSkeleton, Card, VStack } from '@suite-native/atoms';
+import { ReceiveStackParamList, ReceiveStackRoutes, Screen } from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
-import { useAccountReceiveAddress } from '../hooks/useAccountReceiveAddress';
-import { ReceiveAddressCard } from '../components/ReceiveAddressCard';
-import { ReceiveAccountDetailsCard } from '../components/ReceiveAccountDetailsCard';
-import { hasReceiveAddressButtonRequest } from '../hooks/receiveSelectors';
+import { ReceiveAddressScreen } from './ReceiveAddressScreen';
 import { ReceiveScreenHeader } from '../components/ReceiveScreenHeader';
 
-type AccountReceiveProps = {
-    accountKey: AccountKey;
-    tokenContract?: TokenAddress;
-};
+const SCREEN_WIDTH = getScreenWidth();
 
-export const ReceiveAccountScreen = ({ accountKey, tokenContract }: AccountReceiveProps) => {
-    const dispatch = useDispatch();
+const cardStyle = prepareNativeStyle(utils => ({
+    padding: utils.spacings.sp8,
+}));
 
-    const account = useSelector((state: AccountsRootState) =>
-        selectAccountByKey(state, accountKey),
-    );
-    const device = useSelector(selectSelectedDevice);
-    const hasReceiveButtonRequest = useSelector(hasReceiveAddressButtonRequest);
+const ReceiveAccountLoader = () => {
```

Comment:
When can I see this loader?

Reply:
Not sure, might be dead code. Will discuss with the team and create a follow-up PR if necessary.